### DISCUSSION TITLE
Seasonal background not working ?

### DISCUSSION THREAD

**Discussion by MikuOgiwara** [ppy/osu#28219] [Ideas]
In Lazer settings, if you select seasonal backgrounds as "always," they are not activated because the "background sources" setting takes precedence. Additionally, it is not possible to select seasonal backgrounds as a background source.

Would it be possible to enable the seasonal backgrounds option to override the background sources setting, or to include seasonal backgrounds as a selectable option in the background sources?
![331860647-024201d1-64b5-486b-b696-90efd7f7edc8](https://github.com/ppy/osu/assets/108834620/6f860279-54b0-443c-b46c-e481f574c154)

**Comment by SupDos** [comment]
Seasonal background do take precedence over all 3 background source settings so it is supposed to work as you want it to, not sure why yours isn't taking precedence though...

![image](https://github.com/ppy/osu/assets/6813986/6b66f189-932e-4d88-bcdf-2d57a2ba7eb8)


Maybe the game somehow hasn't been able to fetch and download the seasonal backgrounds somehow? If you upload your logs here it might help someone figure it out
